src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
```

Comment:
This image already has height and width. Remove it

Reply:
done